R/pkg/R/SQLContext.R

```diff
@@ -298,6 +298,8 @@ parquetFile <- function(sqlContext, ...) {
 #' Create a SparkDataFrame from a text file.
 #'
 #' Loads a text file and returns a SparkDataFrame with a single string column named "value".
+#' If the directory structure of the text files contains partitioning information, those are
+#' ignored in the resulting DataFrame.
```

Comment:
Say explicitly to use `spark.read.format("text")` if you want to include partition columns when present?